packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -778,22 +780,21 @@ const ValidationEngine = {
   },
 
   registerValidatorInGroup(group, validator) {
-    const groupConfig = ValidationEngine.addGroup(group);
+    const groupConfig = ValidationEngine.addGroup(group, false);
     groupConfig.registerValidator.call(groupConfig, validator);
   },
 
-  _shouldRemoveGroup(group, validatorsInGroup) {
+  _shouldRemoveGroup(group, validators, isRemovable) {
     const isDefaultGroup = group === undefined;
```

Comment:
[missed refactoring]
Now we can skip this special checks for default group, we can just set `isRemovable=false` when we create it

Reply:
It seems that removing the check for default group breaks some test.
![image](https://github.com/DevExpress/DevExtreme/assets/133762589/1d2c2a3e-3c84-4534-b644-1daf17d12129)

Reply:
Update: Already fixed it, seems to be flaky only.

Reply:
[bcf7ab2](https://github.com/DevExpress/DevExtreme/pull/27435/commits/bcf7ab24f7d0f52b0117df135f481cd21f72b335)